lib/rspec/sidekiq/batch.rb

```diff
@@ -70,8 +70,8 @@ def total
         allow(Sidekiq::Batch).to receive(:new)  { RSpec::Sidekiq::NullBatch.new }
         allow(Sidekiq::Batch::Status).to receive(:new)  { RSpec::Sidekiq::NullStatus.new }
       elsif respond_to?(:flexmock)
-        flexmock(Sidekiq::Batch).should_receive(:new) { RSpec::Sidekiq::NullBatch.new }
-        flexmock(Sidekiq::Batch::Status).should_receive(:new) { RSpec::Sidekiq::NullStatus.new }
+        flexmock(Sidekiq::Batch).should_receive(:new).responds(RSpec::Sidekiq::NullBatch.new)
```

Comment:
I think it should be `and_return` (possibly with a block), I cannot find `responds` in flexmock's documentation. https://github.com/jimweirich/flexmock#expectation-actions

Reply:
Ok! `returns` is an alias for `and_return`.